MonitorizareVot/DarkModeColorSchema.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  DarkModeColorSchema.swift
+//  MonitorizareVot
+//
+//  Created by Alex Ioja-Yang on 21/11/2019.
+//  Copyright © 2019 Code4Ro. All rights reserved.
+//
+
+import UIKit
+
+struct DarkModeColorSchema: ColorSchema {
+    static let navigationBarBackground = UIColor(hexCode: 0xFFCC00)
+    static let navigationBarTint = UIColor(hexCode: 0x333E48)
+    
+    // Important: We need to make these checks until we have the minimum deployment set to iOS 11
```

Comment:
I've copied this around to both Schemas, as I'm not sure what exactly it refers to and though there's something I'm missing